playground/Directory.Build.targets

```diff
@@ -1,11 +1,18 @@
 <Project>
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
 
-  <ItemGroup Condition="'$(BuildForTest)' != 'true' and '$(IsAspireHost)' == 'true' and '$(SkipDashboardProjectReference)' != 'true'">
+  <PropertyGroup>
+    <IncludeTestPackages Condition="'$(IncludeTestPackages)' == '' and ('$(IsTestProject)' == 'true' or '$(IsTestSupportProject)' == 'true') and '$(SkipDashboardProjectReference)' == 'true'">true</IncludeTestPackages>
```

Comment:
When is `'$(IsTestProject)' == 'true' or '$(IsTestSupportProject)' == 'true'` true for these playground projects?

Reply:
The `SkipDashboardProjectReference` check here is a little surprising. Why that check?

Reply:
Actually, why do we need to set `IncludeTestPackages` here at all?